src/dsl/word/Always.hpp

```diff
@@ -128,13 +128,16 @@ namespace dsl {
             static std::unique_ptr<threading::ReactionTask> make_idle_task(
                 const std::shared_ptr<threading::Reaction>& reaction) {
 
-                auto idle_task = std::make_unique<threading::ReactionTask>(
-                    reaction,
-                    false,
-                    [](threading::ReactionTask& task) { return DSL::priority(task) - 1; },
-                    DSL::run_inline,
-                    DSL::pool,
-                    DSL::group);
+                auto idle_task = std::make_unique<threading::ReactionTask>(reaction,
+                                                                           false,
+                                                                           DSL::priority,
+                                                                           DSL::run_inline,
+                                                                           DSL::pool,
+                                                                           DSL::group);
+
+
+                // Manipulate the id so it always runs after the always task
+                idle_task->id = std::numeric_limits<NUClear::id_t>::max();
```

Comment:
I had a thought to make the priority enum values even, then have the odd do this.
But setting the id to max also works.

Reply:
It feels a little hacky to do it like this, but at the same time it shouldn't cause a problem